src/example_ova_parameters.json

```diff
@@ -0,0 +1,12 @@
+{ "ovf_cpu_count": "2",
+  "ovf_memory_mb": "4096",
+  "vsphere_image_name": "Red Hat CoreOS",
+  "vsphere_product_name": "Red Hat CoreOS",
+  "vsphere_product_vendor_name": "Red Hat Inc.",
+  "vsphere_product_version": "4.0.0",
+  "vsphere_virtual_system_type": "vmx-07 vmx-08",
+  "vsphere_nested_virt": "false",
+  "vsphere_os_type": "rhel6_64Guest",
```

Comment:
Is this a vmware-ism?

Reply:
The first one is an actual "Product Section" field that shows up in the vSphere UI along side the "Red Hat CoreOS" KV-type pairs.

The last three are vmware-isms.
First one essentially selects from a range of virtual hardware types/generations.
The second one is not actually used here, I'll remove it :-)
The third one is another hint to vmware about the guest.  I'll change this to rhel7.

Ultimately, given the way these scripts are used within assembler, it would probably be nice to make things like the name/version/os-type strings CLI options.  A TODO perhaps.